Exercises/numbers/numbers.js

```diff
@@ -1,3 +1,19 @@
-function extractNumbers (text) {
-  return null;
-}
\ No newline at end of file
+// Task #2
+function extractNumbers(text) {
+  result = parseInt(text);
```

Comment:
It's a bad practice to assign a variable without `let/const` keyword.
Because if you mistype the variable name no error will be shown but a new variable will be created.
Also the `result` variable will become global and will be also available through `window` object which can cause problems when your project is large.